src/components/EResourceSections/PCIInfo.js

```diff
@@ -87,6 +87,15 @@ export default class PCIInfo extends React.Component {
             </KeyValue>
           </Col>
         </Row>
+        <Row>
```

Comment:
Can you have this mirror the four-column layout in the edit page?

Reply:
Done